src/CurrencyNetwork.ts

```diff
@@ -10,21 +11,21 @@ export class CurrencyNetwork {
   }
 
   public getInfo (networkAddress: string): Promise<any> {
-    if (!this.utils.checkAddress(networkAddress)) {
+    if (!ethUtils.isValidAddress(networkAddress)) {
```

Comment:
Why do you not check for a checksum address here?

Reply:
didnt want to make the check to strict originally.. but yea we should always check for checksum at this point

Reply:
Yes maybe it is to strict. Can you make it similar to this https://github.com/ethereum/eth-utils/blob/master/eth_utils/address.py#L80
So only check for checksum address if there are mixed case letter? if all are lower case or all are upper case it was probably not intended to get checked for a checksum.